README.md

```diff
@@ -30,3 +30,45 @@ See the `README.md` under each module's subfolder for a list of supported inputs
 You can control the version of a module dependency by adding `?ref=TAG` at the end of the source argument, as shown in the example above. This is highly recommended. You can find a list of available versions [here](https://github.com/entur/terraform-google-memorystore/releases).
 
 Dependency automation tools such as Renovate Bot will be able to discover new releases and suggest updates automatically.
+
+## Contributing
+
+We welcome contributions to this repo. How to contribute:
+
+* Clone this repo, create a branch locally, make changes
+* Run `terraform fmt` to format the Terraform code
+* Run the tests locally (see [Prerequisites](#prerequisites) and [Local testing](#local-testing))
+* Push your branch to the remote repo, create a pull request (PR)
+* The tests are run by GitHub Actions (see [Automatic testing](#automatic-testing))
+* Team Plattform at Entur reviews the PR, approves and merges the changes
+* Team Plattform creates a new release (see [Versioning]](#versioning)
+
+### Prerequisites
+
+* You need to have Terraform installed (version: see [versions.tf](modules/bucket/versions.tf))
+* You need to have Go installed (version >=1.19 because of the `go:build` directive format used in the go files)
+* You need to have gcloud installed and be authenticated (`gcloud auth application-default login`)
+* Your user must have the necessary permissions on the GCP project used to run the tests
+
+### Versioning
+
+This repository is versioned. Please see GitHub´s [documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) describing how to create a new release.
```

Comment:
Versioning and release workflows are automated using Release Please, perhaps we should point to their docs?
https://github.com/googleapis/release-please#release-please